software/controller/lib/core/comms.h

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 #define COMMS_H
 
 #include "network_protocol.pb.h"
+#include "units.h"
```

Comment:
Probably unnecessary now

Reply:
I think for returning Time "#include "units.h"" is needed. compile threw error twice earlier.